packages/web/src/scss/components/Modal/_Modal.scss

```diff
@@ -30,47 +32,34 @@
         background-color: transparent;
     }
 
-    // 2.
-    &::before {
```

Comment:
I removed the backdrop and used Modal background instead so the backdrop does not cover the scrollbar.